client/me/social-login/service.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import React from 'react';
+import { find, get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import { getCurrentUser } from 'state/current-user/selectors';
+import SocialLoginActionButton from './action-button';
+
+const SocialLoginService = ( { service, icon, isConnected, socialConnectionEmail } ) => (
+	<CompactCard>
+		<div className="social-login__header">
+			<div className="social-login__header-info">
+				<div className="social-login__header-icon">{ icon }</div>
+				<h3 style={ { textTransform: 'capitalize' } }>{ service }</h3>
```

Comment:
Shouldn't we use add that rule in `style.scss` rather than inlining it?

Reply:
Yes, it was actually already there :)